Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -85,7 +85,7 @@ void DSPEmitter::ClearIRAMandDSPJITCodespaceReset()
   CompileDispatcher();
   m_stub_entry_point = CompileStub();
 
-  for (int i = 0x0000; i < 0x10000; i++)
+  for (size_t i = 0; i < MAX_BLOCKS; i++)
```

Comment:
Wouldn't this be a nice place to take advantage of `std::array` by using `.size()`  instead?

Also, it looks a little odd that `m_block`, `m_block_links` and `m_block_size` are `std::vector` (which might be of arbitrary size) but we still work with the same range here as if they were an array of `MAX_BLOCKS` entries. Would there be an advantage (size, performance or whatever) in making them `std::array` too?